src/scanoss/scanossgrpc.py

```
@@ -50,10 +57,13 @@
 from .api.components.v2.scanoss_components_pb2_grpc import ComponentsStub
 from .api.cryptography.v2.scanoss_cryptography_pb2 import AlgorithmResponse
 from .api.cryptography.v2.scanoss_cryptography_pb2_grpc import CryptographyStub
-from .api.dependencies.v2.scanoss_dependencies_pb2 import DependencyRequest
+from .api.dependencies.v2.scanoss_dependencies_pb2 import (
+    DependencyRequest,
+    DependencyResponse,
```

Ruff (0.8.2)

`.api.dependencies.v2.scanoss_dependencies_pb2.DependencyResponse` imported but unused

Remove unused import: `.api.dependencies.v2.scanoss_dependencies_pb2.DependencyResponse`

(F401)